### PR TITLE
zh-translation: /zh/about/feature-stages/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/about/feature-stages/index.md [hunk 1]
```diff
@@ -10,99 +10,92 @@ aliases:
...
-| Resilience features: timeouts, retries, connection pools, outlier detection | Stable
-| Gateway: Ingress, Egress for all protocols | Stable
-| TLS termination and SNI Support in Gateways | Stable
-| SNI (multiple certs) at ingress | Stable
-| [Locality load balancing](/zh/docs/ops/configuration/traffic-management/locality-load-balancing/) | Beta
-| Enabling custom filters in Envoy | Alpha
-| CNI container interface | Alpha
+| 协议: HTTP1.1 / HTTP2 / gRPC / TCP | Stable
+| 协议: Websockets / MongoDB  | Stable
+| 流量控制: 基于标签/内容的路由, 流量切换 | Stable
```
Comment (orangegzx) [inline content/zh/about/feature-stages/index.md:35]:
```suggestion
| 流量控制： 基于标签/内容的路由, 流量切换 | Stable
```

File: content/zh/about/feature-stages/index.md [hunk 2]
```diff
@@ -10,99 +10,92 @@ aliases:
...
+| 谈性功能: 超时， 重试， 连接池，异常检测 | Stable
```
Comment (orangegzx) [inline content/zh/about/feature-stages/index.md:36]:
```suggestion
| 谈性功能：超时， 重试， 连接池，异常检测 | Stable
```

File: content/zh/about/feature-stages/index.md [hunk 3]
```diff
@@ -10,99 +10,92 @@ aliases:
...
+| 网关: 所有协议的出口入口 | Stable
```
Comment (orangegzx) [inline content/zh/about/feature-stages/index.md:37]:
```suggestion
| 协议：HTTP1.1 / HTTP2 / gRPC / TCP | Stable
| 协议：Websockets / MongoDB  | Stable
| 流量控制：基于标签/内容的路由, 流量切换 | Stable
| 谈性功能：超时， 重试， 连接池，异常检测 | Stable
| 网关：所有协议的出口入口 | Stable
```

**Review by kebe7jun — COMMENTED** [review]
我看你这好像不是最新版？
你看看。

**Comment by MXuDong** [comment]
已经更新 @kebe7jun ,中文原文没有更新，我忘记同步 英文了

File: content/zh/about/feature-stages/index.md [hunk 4]
```diff
@@ -10,99 +10,90 @@ aliases:
...
-| Consul Integration | Alpha
-| Basic Configuration Resource Validation | Beta
-| Configuration Processing with Galley | Beta
-| [Mixer Self Monitoring](/zh/faq/mixer/#mixer-self-monitoring) | Beta
-| [Custom Mixer Build Model](https://github.com/istio/istio/wiki/Mixer-Compiled-In-Adapter-Dev-Guide) | deprecated
-| [Out of Process Mixer Adapters (gRPC Adapters)](https://github.com/istio/istio/wiki/Mixer-Out-Of-Process-Adapter-Dev-Guide) | Beta
-| [Istio CNI plugin](/zh/docs/setup/additional-setup/cni/) | Alpha
-| IPv6 support for Kubernetes | Alpha
-| [Distroless base images for Istio](/zh/docs/ops/configuration/security/harden-docker-images/) | Alpha
+| [标准操作器](/zh/docs/setup/install/standalone-operator/) | Beta
```
Comment (kebe7jun) [inline content/zh/about/feature-stages/index.md:84]: operator 不翻译。

File: content/zh/about/feature-stages/index.md [hunk 5]
```diff
@@ -10,99 +10,90 @@ aliases:
...
-| TLS termination and SNI Support in Gateways | Stable
-| SNI (multiple certs) at ingress | Stable
-| [Locality load balancing](/zh/docs/ops/configuration/traffic-management/locality-load-balancing/) | Beta
-| Enabling custom filters in Envoy | Alpha
-| CNI container interface | Alpha
+| 协议: HTTP1.1 / HTTP2 / gRPC / TCP | Stable
+| 协议: Websockets / MongoDB  | Stable
+| 流量控制: 基于标签/内容的路由, 流量切换 | Stable
+| 谈性功能: 超时， 重试， 连接池，异常检测 | Stable
+| 网关: 所有协议的出口入口 | Stable
```
Comment (kebe7jun) [inline content/zh/about/feature-stages/index.md:38]: 出入口网关不翻译吧。

**Review by kebe7jun — COMMENTED** [review]
解决下 lint。

**Comment by kebe7jun** [comment]
修复一下 lint . @MXuDong

**Review by Xunzhuo — COMMENTED** [review]
fix below:
./content/zh/about/feature-stages/index.md:58:| [链路追踪样本](/docs/tasks/observability/distributed-tracing/configurability/#trace-sampling) | Beta
./content/zh/about/feature-stages/index.md:59:| [请求分类](/docs/tasks/observability/metrics/classify-metrics/) | Beta
./content/zh/about/feature-stages/index.md:90:| [外部控制平面](/docs/setup/additional-setup/external-controlplane/) | Alpha
./content/zh/about/feature-stages/index.md:95:| [虚机集成](/docs/setup/install/virtual-machine/) | Beta

**Comment by windsonsea** [comment]
/close 
because this my be oudated 18 months ago.

**Pull request closed without merging**